### DISCUSSION TITLE
Credential Cache Duration

### DISCUSSION THREAD

**Discussion by felgryn** [git-ecosystem/git-credential-manager#1435] [Q&A] [answered]
Hello,

 I've installed and configured GCM, however, i don't seem to be able to get the cache duration working properly.

 I have the following settings from git config:

credential.helper=/usr/local/bin/git-credential-manager
credential.credentialstore=cache
credential.cacheduration=43200

and with the cacheduracton of 43200, it should be 12 hours. However, i find it looks like it is continuing to operate with the 15 minute default. Not sure what else i should be changing to get the cache to extend beyond the default. Any suggestions?

Cheers!

**Comment by hickford** [comment, marked as answer]
Try https://github.com/git-ecosystem/git-credential-manager/blob/main/docs/configuration.md#credentialcacheoptions

    credential.cacheOptions=--timeout 43200

**Reply by felgryn** [reply]
Many thanks @hickford ! That got the timeout working for me.
